src/main/java/frc/robot/Robot.java

```diff
@@ -32,18 +32,39 @@ public void robotInit() {
         shooterSystem = new ShooterSystem();
         intakeSystem = new IntakeSystem();
         armSystem = new ArmSystem();
-        xboxController = new XboxController(0);
+        xboxController = new XboxController(RobotMap.XBOX_CONTROLLER_PORT);
 
-        POVButton dPadUp = new POVButton(xboxController, 0);
-        POVButton dPadDown = new POVButton(xboxController, 180);
+        armSystem.setDefaultCommand(
+                new ParallelCommandGroup(
+                        new ParallelDeadlineGroup(
+                                new WaitUntilCommand(() -> armSystem.reachedATargetAngle(RobotMap.ARM_SHOOTER_ANGLE)),
+                                new InstantCommand(() -> intakeSystem.slowIn())
+                        ),
+                new ArmMoveToShooterCommand(armSystem)
+                )
+        );
+
+        POVButton dPadUp = new POVButton(xboxController, RobotMap.D_PAD_TO_SHOOTER_ANGLE);
+        POVButton dPadDown = new POVButton(xboxController, RobotMap.D_PAD_TO_FLOOR_ANGLE);
 
         dPadUp.onTrue(new ArmMoveToShooterCommand(armSystem));
         dPadDown.onTrue(new ArmMoveToFloorCommand(armSystem));
 
         new JoystickButton(xboxController, XboxController.Button.kX.value).whileTrue(new ShootOut(shooterSystem));
-        new JoystickButton(xboxController, XboxController.Button.kB.value).whileTrue(new ShooterPID(shooterSystem, 2000));
+        //new JoystickButton(xboxController, XboxController.Button.kB.value).whileTrue(new ShooterPID(shooterSystem, RobotMap.SHOOTER_TARGET_RPM));
         new JoystickButton(xboxController, XboxController.Button.kY.value).whileTrue(new OuttakeCommand(intakeSystem));
-        new JoystickButton(xboxController, XboxController.Button.kA.value).whileTrue(new IntakeCommand(intakeSystem));
+        //new JoystickButton(xboxController, XboxController.Button.kA.value).whileTrue(new IntakeCommand(intakeSystem));
+
+        ParallelDeadlineGroup collectNote = new ParallelDeadlineGroup(
```

Comment:
I guess this will work, considering that a note will only enter when the arm is on the floor. But maybe first reach the floor with the arm and only then start `IntakeCommand`?

Reply:
done

Reply:
The command runs first the Intake and them the arm to the floor. Should be arm-to-floor and then intake

Reply:
Done